include/powsybl/iidm/converter/xml/AbstractExtensionXmlSerializer.hpp

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
```

Comment:
2020 ;)

Reply:
Git consider this file as a new file whereas I just move it. I will fix it